internal/packages/archetype/package_test.go

```diff
@@ -52,17 +57,28 @@ func createPackageDescriptorForTest(packageType, kibanaVersion string) PackageDe
 			},
 		}
 	}
+	version := "1.2.3"
 	specVersion, err := GetLatestStableSpecVersion()
+	excludeChecks := []string{}
 	if err != nil {
 		panic(err)
 	}
+	if packageType == "content" {
+		minSpecVersion := semver.MustParse("3.4.0")
+		if !specVersion.LessThan(minSpecVersion) {
+			panic("this code can be removed")
+		}
+		version = "0.1.0"
+		specVersion = *minSpecVersion
+		excludeChecks = append(excludeChecks, "PSR00002")
```

Comment:
Currently, `elastic-package create package` it would create a content package with `format_version: 3.2.2` (latest spec version published). Should it be set somehow here for content packages at least 3.4.0 ? 

Just created a package, and when running `elastic-package build`, it shows these errors:
```
Error: building package failed: invalid content found in built zip package: found 6 validation errors:
   1. package with non-stable semantic version and active beta features (enabled in [/home/user/Coding/work/elastic-package/build/packages/new_package-0.0.1.zip]) can't be released as stable version. (PSR00002)
   2. item [LICENSE.txt] is not allowed in folder [/home/user/Coding/work/elastic-package/build/packages/new_package-0.0.1.zip]
   3. item [changelog.yml] is not allowed in folder [/home/user/Coding/work/elastic-package/build/packages/new_package-0.0.1.zip]
   4. item [docs] is not allowed in folder [/home/user/Coding/work/elastic-package/build/packages/new_package-0.0.1.zip]
   5. item [img] is not allowed in folder [/home/user/Coding/work/elastic-package/build/packages/new_package-0.0.1.zip]
   6. item [manifest.yml] is not allowed in folder [/home/user/Coding/work/elastic-package/build/packages/new_package-0.0.1.zip]
```

When updating the format_version to 3.4.0, it still reports this error:
```
Error: building package failed: invalid content found in built zip package: found 1 validation error:
   1. package with non-stable semantic version and active beta features (enabled in [/home/mariorodriguez/Coding/work/elastic-package/build/packages/new_package-0.0.1.zip]) can't be released as stable version. (PSR00002)
```

Reply:
These are handled in the test case, I didn't add the exceptions to the real command to don't make it too easy to use content packages yet, and to avoid us having to remember to remove these exceptions when/if the package is released as GA.

Would you prefer that we add the exceptions in the command?

Reply:
Ah ok! Better to force to add the exceptions manually. No need to change anything then.